teamplate.rb

```diff
@@ -18,11 +18,12 @@
 generate "bootstrap:install", "static"
 generate "bootstrap:layout" # auto overwrite
 
+# Figaro
 # Other gems
-%w( quiet_assets pg ).each { |name| gem name }
+%w( quiet_assets pg rails-erd figaro).each { |name| gem name }
```

Comment:
Check http://guides.rubyonrails.org/rails_application_templates.html - it looks like you can `run "command"`. You may need to make sure that that happens after the bundle install runs though.